lib/reviewit/lib/reviewit/action/push.rb

```diff
@@ -48,6 +51,16 @@ def append_mr_id_to_commit mr_id
       end
     end
 
+    def rename_local_branch mr_id
+      branch = options[:branch]
```

Comment:
The idea is to use the local branch name, not the target branch.

Because many times the target branch is just a version number that says not much about what's in there.

So instead of use `options[:branch]`, better to use:
```
`git symbolic-ref -q HEAD`.gsub('refs/heads/', '').strip
```